projects/packages/wp-js-data-sync/package.json

```diff
@@ -1,29 +1,29 @@
 {
```

Comment:
I don't know how this file got checked in with bad formatting. `tools/fixup-project-versions.sh` fixed it, and I decided to leave the reformatting in this PR to get it cleaned up.